docs/setup-and-configuration.md

```diff
@@ -11,5 +11,486 @@ A client is usually some system that you want to able to send request to the Ope
 
```

Comment:
Please remove the blank line at the beginning of this file - it should be the main heading

Reply:
I missed it on previous review